DNN Platform/Library/UI/Skins/SkinPackageInfo.cs

```diff
@@ -79,8 +92,10 @@ public string SkinName
             }
         }
 
-        [XmlIgnore]
+        /// <summary>Gets or sets a dictionary mapping from <see cref="SkinInfo.SkinId"/> to <see cref="SkinInfo.SkinSrc"/>.</summary>
+        [XmlIgnore]
         [JsonIgnore]
+        [Obsolete($"Deprecated in DotNetNuke 9.13.1. Use {nameof(ISkinPackageInfo)}.{nameof(ISkinPackageInfo.Skins)} instead. Scheduled for removal in v11.0.0.")]
```

Comment:
Note, the SkinController only updates the `SkinsList` (`skinPackage` is `ISkinPackageInfo` here):

https://github.com/dnnsoftware/Dnn.Platform/blob/2d859732a381bfcdea259e3a5274b44c25a3566b/DNN%20Platform/Library/UI/Skins/SkinController.cs#L641-L656

Even though the property is deprecated, it's read-only now.

Reply:
We currently have two properties:

`Skins` = Dictionary<int, string>
`SkinsList` = List<ISkinInfo>

If you change the SkinsList and call UpdateSkinPackage the skins will be updated.
However, if you change the Skins and call UpdateSkinPackage the skins will not be updated, which happens in the current version.

> Or that we're missing an update to keep the two collection in sync?

Correct.